bundles/org.openhab.binding.haywardomnilogic/src/main/resources/ESH-INF/thing/backyard.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="haywardomnilogic"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="backyard" listed="false">
```

Comment:
I didn't see any reason to list them as you OpenHab user can't effectively add them as a thing unless they exist in the Hayward OmniLogix pool controller, in which case it will be automatically discovered.  With that being said, I know it's frowned upon to remove child things of a bridge, but in this case, if you delete the bridge, the child things are abandoned and must be force removed.  If the bridge is re-added, it will discover all of the children things.  Knowing this, can I automatically remove the children if the bridge is removed?  If so, what is the method used to do so?  Thx.

Reply:
Ok. seems a logical reason not have them listed.
It's not possible to remove children from bindings. I would say the force remove is more a limitation the ui/way things are removed than that it should be a concern for the binding. If a bridge would be re-added I would expect that all existing things are not rediscovered, but might be needed to reconfigured by editing the thing and setting the bridge again. If a thing would have been manually created and the bridge would have the same uid than re-adding the brigde would simply enable the things again.

Reply:
I like the idea of updating orphaned things to point at a new bridge connection.  I can store the MSPID in each thing to ensure the orphaned things don't get consumed by a new bridge connection that belongs to a different pool/property and therefore a different MSPID.  What is the easiest way to locate, iterate through and modify orphaned things?  Are there any examples of this laying around?  Thanks.

Reply:
One caveat.  The linked items will need their channel's updated as well since they contain the bridgeID.  Right?  If so, any direction on accessing and modifying that information would be appreciated.  One issue.  If the items are hard coded in an items file, of course those will be broken, but that is to be expected.